README.md

```diff
@@ -35,6 +35,15 @@ issue appeals to you, feel free to jump in and get started on it.
 However, there's a bunch of design stuff that also needs to happen, but we'll
 make sure to have the conversation in an open manner.
 
+Where/how can I report vulnerabilities?
+---------------------------------------
+
+Please see our [this
+document](https://github.com/quictls/quictls/blob/main/SECURITY.md)
```

Comment:
Should 'this document' be 'security policy'? Or spell it out fully: 'security policies or procedures'? Or (worst option in my opinion) drop 'our'.

Reply:
Good call. :-)

https://github.com/quictls/quictls/pull/54